torch/_torch_docs.py

```diff
@@ -1903,6 +1903,53 @@ def merge_dicts(*dicts):
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.copysign,
+           r"""
+copysign(input, other, *, out=None) -> Tensor
+
+Create a new float tensor with the magnitude of :attr:`input` and the sign of :attr:`other`, elementwise.
```

Comment:
nit: "float" is a bit ambiguous here (is it float32 or floating-point in general?). I think "floating-point" would clarify this more here